src/js/components/icons/status/Disabled.js

```diff
@@ -9,12 +10,19 @@ var Disabled = React.createClass({
 
   render: function() {
     var className = 'status-icon status-icon-disabled';
+    var a11yTitle;
     if (this.props.className) {
       className += ' ' + this.props.className;
     }
+    if (typeof this.props.a11yTitle === "undefined") {
```

Comment:
I think we can replace this by:

`a11yTitle = this.props.a11yTitle || this.getGrommetIntlMessage('Disabled')`